api-spec.md

```diff
@@ -125,33 +63,23 @@ Local Look-up
 #### Body Example:
 ```
 {
-  "device_id": "xxxxxx"
+  "fromDate": "timestamp",
```

Comment:
Good touch with `fromDate`, I actually forgot that we need it.

Reply:
lets have the specification for `date` and `location` in the query params for this route, initial GET can specify the location, and any other subsequent GETs after the first request can include a specification for the `fromDate` as well, what do you think of this? any other params you'd need?